packages/muelu/src/Utils/MueLu_AlgebraicPermutationStrategy_decl.hpp

```diff
@@ -90,9 +90,12 @@ void sortingPermutation(const std::vector<Scalar> & values, std::vector<LocalOrd
      * \param A: input matrix (input)
      * \param permRowMap: Dof row map permutation shall be restricted on (input)
      * \param currentLevel: only for output of variables
-     * \param genFactory: const pointer to generating (calling) PermutationFactory // TODO avoid this, not very elegant. Decide which variables have to be generated, give them back per reference to the PermutationFactory.
+     * \param genFactory: const pointer to generating (calling) PermutationFactory
+     // TODO avoid this, not very elegant.
```

Comment:
Not sure how Doxygen feels about these `//` comments interrupting a documentation block, but that's for a later fix.